packages/common/http/src/request.ts

```diff
@@ -119,6 +119,11 @@ export class HttpRequest<T> {
 
   /**
    * Outgoing URL parameters.
+   *
+   * To pass a string representation of HTTP parameters in the URL-query-string format,
+   * the `HttpParamsOptions`' `fromString` may be used.
+   *
+   * Example; `new HttpParams({fromString: 'angular=awesome'})`
```

Comment:
```suggestion
   * the `HttpParamsOptions`' `fromString` may be used. For example:
   *
   * ```
   * new HttpParams({fromString: 'angular=awesome'})
   * ```
```

Reply:
Not sure how that will look in the preview but I think it is cleaner.

Reply:
Looks pretty good, IMO. Just pushed...
![image](https://user-images.githubusercontent.com/61225472/109298267-e42b3900-7865-11eb-9486-07455e47ad17.png)

Reply:
👍